modules/composer-ffmpeg/src/main/java/org/opencastproject/composer/impl/EncoderEngine.java

```diff
@@ -58,7 +56,7 @@
 /**
  * Abstract base class for encoder engines.
  */
-public class EncoderEngine implements AutoCloseable {
+public class encoderEngine implements AutoCloseable {
```

Comment:
Shouldn't this be capitalised?  I'm surprised checkstyles isn't freaking out right now...

Reply:
That's because it current is freaking out...  Does not build due to checkstyles failing this.